### DISCUSSION TITLE
Route 53 shows only 2 records, & not showing DNS entries for frontend, backend, mysql components

### DISCUSSION THREAD

**Discussion by pjuned** [learndevopsonline/learndevopsonline#1560] [Q&A]
HI Raghu,

Route 53 shows only 2 records, & not showing DNS entries for frontend, backend, mysql components

please check screenshot

Thanks
![dns](https://github.com/learndevopsonline/learndevopsonline/assets/148682494/65045d63-496b-4d1d-b0bf-8d3707148fdd)

**Comment by sreedharm07** [comment]
Hi @pjuned you should add that frontend , backend , mysql records

**Reply by sreedharm07** [reply]
You see create record there
1- click on that 
2- give your ip 
3- give your frontend domain name 
4- A record 
5- create

Do same to 3 components

**Reply by pjuned** [reply]
2) give your ip means , the public/private instance ip of the components  ?

**Reply by sreedharm07** [reply]
Private ips of components
